astropy/cosmology/io/tests/test_mapping.py

```diff
@@ -129,6 +129,36 @@ def test_tofrom_mapping_move_tofrom_meta(self, cosmo, to_format, from_format):
         assert got == cosmo  # (Doesn't check metadata)
         assert got.meta["mismatching"] == "will error"
 
+    def test_to_mapping_rename_conflict(self, cosmo, to_format):
+        """Test ``rename`` in ``to_mapping()``."""
+        to_rename = {"name": "name"}
+        with pytest.raises(ValueError, match="'renames' values must be disjoint"):
```

Comment:
Here the `match` value is clear enough to convey the purpose of the test, but we also want the error message to tell the user what the conflicting names are, so we should test that by expanding the `match` value even more.
```suggestion
        with pytest.raises(ValueError, match="'renames' values must be disjoint"):
```

Reply:
Done!